docs: onboarding users (#7462)
zitadel/zitadel: docs/docs/guides/integrate/onboarding/end-users.mdx

### Built-in register form

#### Prompt registration form

When users first access your application, they'll typically need to either log in or register.
By default, redirecting from your application displays the login page.

> OpenID Connect (OIDC) allows you to control the initial screen by sending a [prompt](/docs/apis/openidoauth/endpoints#additional-parameters) in the authorization request.

Reviewer: ```suggestion
OpenID Connect (OIDC) allows you to control the initial screen by sending a [prompt](/docs/apis/openidoauth/endpoints#additional-parameters) parameter in the authorization request.
```

> With the `prompt=create` the registration form/options will directly be shown to the user.

Reviewer: ```suggestion
With the `prompt=create`, the registration form/options will directly be shown to the user.
```

You can test the impact of the different prompts on your login UI in our [OIDC Playground](/docs/apis/openidoauth/authrequest).

Unfortunately, SAML doesn't offer the same level of control over the initial screen.
You won't be able to directly influence which page (login or registration) is shown through the SAML flow.

#### Choose Registration Option

> If an organization does allow local user registration as well as registration with a social or enterprise login, the user will have to choose the option.

Reviewer: ```suggestion
If an organization allows local user registration as well as registration with a social or enterprise login, the user can choose an option to register.
```

> As soon as the user clicks the "register" button, the registration options screen will be shown.

Reviewer: ```suggestion
As soon as users click the "register" button, they will be presented with a screen showing the different registration options.
```

<img src="/docs/img/guides/solution-scenarios/register-options.png" alt="Register Options"/>

> The use can now select either local user registration or an external provider.

Reviewer: ```suggestion
After that, the user can select either local user registration or an external provider.
```

> By pressing the button of an external provider, the user will directly be redirected to the provider.

Reviewer: ```suggestion
By pressing the button of an external provider, the user will directly be redirected to the provider for consent.
```
